lang/rust/avro_derive/tests/derive.rs

```diff
@@ -1063,4 +1063,40 @@ mod test_derive {
         serde_assert(TestBasicWithU32 { a: u32::MIN });
         serde_assert(TestBasicWithU32 { a: 1_u32 });
     }
+
+    #[derive(Debug, Serialize, Deserialize, AvroSchema, Clone, PartialEq)]
+    #[avro(alias = "a", alias = "b", alias = "c")]
```

Comment:
I also wonder if this would work with:
```
#[avro(alias = "a")]
#[avro(alias = "b")]
#[avro(alias = "c")]
```

Reply:
It does - https://github.com/apache/avro/pull/1647/commits/05cf7807c947a58a6ab76aa50dfa8bb2a3999296